pkgs/top-level/all-packages.nix

```diff
@@ -24962,6 +24962,10 @@ in
 
   rfc-bibtex = python3Packages.callPackage ../development/python-modules/rfc-bibtex { };
 
+  pick-colour-picker = python3Packages.callPackage ../development/python-modules/pick-colour-picker {
```

Comment:
Programs generally should not be in `python-modules` but rather the most appropriate directory based on the category of the software.